### PR TITLE
[incubator/elasticsearch] Add pod disruption budgets.

### PR DESCRIPTION
**Pull request by icereval**
Updated from https://github.com/kubernetes/charts/pull/2530

### CODE REVIEW

**Comment by icereval** [comment]
Add PDBs @ https://github.com/kubernetes/charts/issues/2782

File: incubator/elasticsearch/README.md [hunk 1]
```diff
@@ -69,30 +69,35 @@ The following tables lists the configurable parameters of the elasticsearch char
...
 | `data.persistence.name`              | Data statefulset PVC template name                                  | `data`                               |
 | `data.persistence.size`              | Data persistent volume size                                         | `30Gi`                               |
 | `data.persistence.storageClass`      | Data persistent volume Class                                        | `nil`                                |
 | `data.persistence.accessMode`        | Data persistent Access Mode                                         | `ReadWriteOnce`                      |
-| `data.podAnnotations`                | Data StatefulSet annotations                                        | `{}`                                 |
 | `data.terminationGracePeriodSeconds` | Data termination grace period (seconds)                             | `3600`                               |
-| `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                               |
```
Comment (paulczar) [inline incubator/elasticsearch/README.md:95]: it looks like there's a few documentation fixups that are unrelated to the PDBs ...  It might be a good idea to put them through as a seperate PR to reduce the scope of this PR to just the PDBs.

**Review by paulczar — CHANGES_REQUESTED** [review]
it looks like there's a few documentation fixups that are unrelated to the PDBs around podAnnotations and antiAffinity.

It might be a good idea to put them through as a seperate PR to reduce the scope of this PR to just the PDBs.
